.github/actions/inductor-xpu-e2e-test/action.yml

```diff
@@ -57,6 +57,11 @@ runs:
           rm -rf benchmark && git clone https://github.com/pytorch/benchmark.git
           cd benchmark && git checkout $TORCHBENCH_COMMIT_ID && pip install --no-deps -r requirements.txt
           python install.py --continue_on_fail
+          # deps for torchrec_dlrm
```

Comment:
Why xpu need those deps? Can this model run on xpu with those deps?

Reply:
Yes, torchrec_dlrm will be passed for FP32 if install the related deps